doc/examples/denoise_mppca.py

```diff
@@ -0,0 +1,282 @@
+"""
+======================================================
+Denoise images using the Marcenko-Pastur PCA algorithm
+======================================================
+
+The PCA-based denoising algorithm exploits the redundancy across the
+diffusion-weighted images [Manjon2013]_, [Veraart2016a]_. This algorithm has
+been shown to provide an optimal compromise between noise suppression and loss
+of anatomical information for different techniques such as DTI [Manjon2013]_,
+spherical deconvolution [Veraart2016a] and DKI [Henri2018]_.
+
+The basic idea behind the PCA-based denoising algorithm is to remove the
```

Comment:
The basic idea behind the PCA-based denoising algorithms is to remove the
Principal Components of the data that are mostly classified as noise.